app-i18n/fcitx-configtool/fcitx-configtool-5.1.2.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_PN="fcitx5-configtool"
+S="${WORKDIR}/${MY_PN}-${PV}"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}.tar.xz -> ${P}.tar.xz"
+KEYWORDS="~amd64 ~arm64 ~loong ~x86"
```

Comment:
If I understood correctly people in gentoo-zh have tested all of these with all of these packages?

Would have to hear opinions on if merging them with keywords already in place is ok or if rekeywording should be insisted on.